.github/workflows/welcome-first-time-contrib.yml

```diff
@@ -0,0 +1,13 @@
+name: Greeting the new contributor 
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Welcome to offical website repository for [json-schema.org](https://json-schema.org/). Thanks a lot for reporting your first issue. Please check out our [contributors guide](https://github.com/json-schema-org/website/blob/main/CONTRIBUTING.md) useful for opening a pull request.<br /> For more details check out [README.md](https://github.com/json-schema-org/website?tab=readme-ov-file#-welcome-to-the-json-schema-website) file.'
+        pr-message: 'Welcome to offical repository for [json-schema.org](https://json-schema.org/). Thanks a lot for creating your first pull request. Please check out our [contributors guide](https://github.com/json-schema-org/website/blob/main/CONTRIBUTING.md) useful for opening a pull request.<br /> For more details check out [README.md](https://github.com/json-schema-org/website?tab=readme-ov-file#-welcome-to-the-json-schema-website) file.'
```

Comment:
> Please check out our contributors guide if you plan on opening a pull request.

In this case, someone opening an issue may just be reporting something without any intention of fixing it themselves.